Add max_docs_per_value to diversified sampler documentation (#6134)
opensearch-project/documentation-website: _aggregations/bucket/diversified-sampler.md

  - /query-dsl/aggregations/bucket/diversified-sampler/
---

# Diversified sampler

> The `diversified_sampler` aggregation lets you reduce the bias in the distribution of the sample pool by de-duplicating documents with the same `field`. It does so by using the `max_docs_per_value` and `field` settings, which limit the maximum number of documents collected on a shard for the provided `field`. The `max_docs_per_value` setting is an optional parameter used to determine up to how many documents per `field` will be returned. The default value of this setting is `1`.

Reviewer: ```suggestion
The `diversified_sampler` aggregation lets you reduce the bias in the distribution of the sample pool by deduplicating documents containing the same `field`. It does so by using the `max_docs_per_value` and `field` settings, which limit the maximum number of documents collected on a shard for the provided `field`. The `max_docs_per_value` setting is an optional parameter used to determine up to how many documents per `field` will be returned. The default value of this setting is `1`.
```

> The `diversified_sampler` aggregation lets you reduce the bias in the distribution of the sample pool by de-duplicating documents with the same `field`. It does so by using the `max_docs_per_value` and `field` settings, which limit the maximum number of documents collected on a shard for the provided `field`. The `max_docs_per_value` setting is an optional parameter used to determine up to how many documents per `field` will be returned. The default value of this setting is `1`.

Reviewer: Last sentence: "used to determine the maximum number of documents that will be returned per `field`"?

> Similar to the [`sampler` aggregation]({{site.url}}{{site.baseurl}}/aggregations/bucket/sampler/), you can use the `shard_size` setting to control the maximum number of documents collected on any one shard, as shown in the following example:

Reviewer: ```suggestion
Similarly to the [`sampler` aggregation]({{site.url}}{{site.baseurl}}/aggregations/bucket/sampler/), you can use the `shard_size` setting to control the maximum number of documents collected on any one shard, as shown in the following example:
```

# Diversified sampler aggregations

> The `diversified_sampler` aggregation lets you reduce the bias in the distribution of the sample pool by de-duplicating documents with the same `field`. It does so via the `max_docs_per_value` and `field` settings, which limit the maximum number of docs collected on a shard for the provided `field`. Similar to the Sampler aggregation, you can use the `shard_size` setting to control the maximum number of documents collected on any one shard:

Reviewer 2: ```suggestion
The `diversified_sampler` aggregation lets you reduce the bias in the distribution of the sample pool by de-duplicating documents with the same `field`. It does so by using the `max_docs_per_value` and `field` settings, which limit the maximum number of documents collected on a shard for the provided `field`. Similar to the Sampler aggregation, you can use the `shard_size` setting to control the maximum number of documents collected on any one shard:
```
